jobs/build-publish-image.groovy

```diff
@@ -41,6 +41,9 @@ def runScript() {
    kaGit.safeSyncToOrigin("git@github.com:Khan/webapp",
                           params.GIT_REVISION);
 
+   // Prune docker images before building if under 1.5GB of disk space.
+   sh("[ $(df -BM --output=avail . | tr -cd 0-9) -gt 1500 ] || docker image prune -af")
```

Comment:
Hi @csilvers, we're getting an error on this line when attempting to build a publish worker image: https://jenkins.khanacademy.org/job/deploy/job/build-publish-image/219/console

```
WorkflowScript: 45: illegal string body character after dollar sign;
   solution: either escape a literal dollar sign "\$5" or bracket the value expression "${5}" @ line 45, column 6.
      sh("[ $(df -BM --output=avail . | tr -cd 0-9) -gt 1500 ] || docker image prune -af")
```

Do you know what the fix here is?